src/pages/cli-legacy/graphql-transformer/storage.mdx

```diff
@@ -1,10 +1,12 @@
 export const meta = {
   title: `GraphQL transform and Storage`,
-  description: `The GraphQL Transform, Amplify CLI, and Amplify Library make it simple to add complex object support with Amazon S3 to an application.`,
+  description: `The GraphQL Transform, Amplify CLI, and Amplify Library make it simple to add complex object support with Amazon S3 to an application.`
 };
 
 The GraphQL Transform, Amplify CLI, and Amplify Library make it simple to add complex object support with Amazon S3 to an application.
 
+> **Note:** Complex objects are not supported by DataStore-enabled GraphQL APIs.
```

Comment:
should this be a `<Callout />`?

Reply:
should I make it a warning or just the default?

Reply:
I'd make a warning if this is something that regularly trips folks up. Otherwise, a the default should work.